### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by vivekgrover1**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+        - ./vars.env
      restart: always
 
-   jenkins_bot:
-     container_name: jenkins_slack_bot
-     extends: db_bot
+   jenkins_bot_rtm:
+     container_name: jenkins_slack_bot_rtm
+     extends: db
      entrypoint: python3.4 slackbot.py
```
Comment (dennyzhang) [inline docker-compose.yml:24]: Do we need to configure restart policy?

Comment (vivekgrover1) [reply docker-compose.yml:24]: I think restart policy is in version 3 not in version 2 . This file is written in version 2.

Comment (dennyzhang) [reply docker-compose.yml:24]: @vivekgrover1 , checking v2, restart is supported.
https://docs.docker.com/compose/compose-file/compose-file-v2/#restart

Could you double confirm this?

If v2 really doesn't support it, maybe we should remove L19?

Comment (vivekgrover1) [reply docker-compose.yml:24]: @DennyZhang I have included the restart in all the services.

**Pull request merged**